rpcserver.go

```diff
@@ -3998,83 +3998,14 @@ func (r *rpcServer) IgnoreAssetOutPoint(ctx context.Context,
 		req.AssetAnchorPoint)
 
 	assetID := req.AssetAnchorPoint.ID
+	assetSpec := asset.NewSpecifierFromId(assetID)
```

Comment:
Is there a particular reason not to construct the assetSpec with asset.NewSpecifierOptionalGroupKey as was done before, and then pass *that* assetSpec to the new method?